modules/integration/src/test/scala/scala/cli/integration/RunTestDefinitions.scala

```diff
@@ -520,6 +520,39 @@ abstract class RunTestDefinitions(val scalaVersionOpt: Option[String])
     }
   }
 
+  test("Debugging") {
+    val inputs = TestInputs(
+      os.rel / "Foo.scala" ->
+        s"""object Foo {
+           |  def main(args: Array[String]): Unit = {
+           |    println("foo")
+           |  }
+           |}
+           |""".stripMargin
+    )
+    inputs.fromRoot { root =>
+      val out1 = os.proc(TestUtil.cli, "run", extraOptions, ".", "--debug", "--command")
+        .call(cwd = root).out.trim().split("\n").toList
+      val out2 = os.proc(
+        TestUtil.cli,
+        "run",
+        extraOptions,
+        ".",
+        "--debug-port",
+        "5006",
+        "--debug-mode",
+        "listen",
+        "--command"
+      ).call(cwd = root).out.trim().split("\n").toList
```

Comment:
the following should fix this test for Windows on the CI (splitting by `\n` won't cut it there)
```suggestion
      ).call(cwd = root).out.trim.lines.toList.asScala
```